### ISSUE TITLE
"Missing prebuilt package" error message is missing essential information

### ISSUE THREAD

**Issue by sigiesec** [conan-io/conan#3621]
When running `conan install` for a package requiring "Bar", which depends on "Foo", I got the following error message: 
```
Foo/1.3.0-unreleased-build.1068@xxx/unstable: WARN: Can't find a 'Foo/1.3.0-unreleased-build.1068@xxx/unstable' package for the specified options and settings:
- Settings: arch=x86_64, build_type=Release, compiler=Visual Studio, compiler.version=14, os=Windows
- Options: <irrelevant here, did not change>
- Package ID: 176ce45f6be376c19c830020e1517258f4d47e2f

ERROR: Missing prebuilt package for 'Foo/1.3.0-unreleased-build.1068@xxx/unstable'
Try to build it from sources with "--build Foo"

Or read "http://docs.conan.io/en/latest/faq/troubleshooting.html#error-missing-prebuilt-package"
```

What changed before was that an (unnecessary) dependency of Foo on FooDependency was removed. Maybe this changed the expected Package ID of Foo?

However, this is completely non-obvious from the error message.

I know that the displayed options influence the package id, but if the (transitive) dependencies also influence it, they should also be part of the error/warning message.

**Comment by danimtb** [comment]
This is a duplicated of #3316 that has already been addressed in PR #3438.
It will be fixed in next release 1.8

Thanks for reporting!

**Comment by sigiesec** [comment]
Great! Sorry for me not checking thoroughly for an existing issue.
